src/components/Auths/logoutButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+
+function LogoutButton() {
+  const { logout } = useAuth0();
+  return (
+    <button
```

Comment:
Why are you using plain jsx button?
Also, this component looks very similar to `loginButton`, you should generalize.